examples/Argon/200K_18mol_l/config_autopas_allOptions.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why an extra file for AutoPas? If its called "allOptions" I would expect it to contain all options.

Reply:
^^^^^^^^^^

Reply:
is now updated, the only thing missing was the tuning strategies.
also added the GeneralDomainDecomposition to that file.